cristin-commons/src/main/java/no/unit/nva/cristin/common/handler/CristinQueryHandler.java

```diff
@@ -37,22 +39,32 @@ protected String getValidPage(RequestInfo requestInfo) throws BadRequestExceptio
         if (Utils.isPositiveInteger(page)) {
             return page;
         }
-        throw new BadRequestException(ERROR_MESSAGE_PAGE_VALUE_INVALID);
+        throw new BadRequestException(ErrorMessages.ERROR_MESSAGE_PAGE_VALUE_INVALID);
     }
 
     protected String getValidNumberOfResults(RequestInfo requestInfo) throws BadRequestException {
         String results = getQueryParam(requestInfo, NUMBER_OF_RESULTS).orElse(DEFAULT_NUMBER_OF_RESULTS);
         if (Utils.isPositiveInteger(results)) {
             return results;
         }
-        throw new BadRequestException(ERROR_MESSAGE_NUMBER_OF_RESULTS_VALUE_INVALID);
+        throw new BadRequestException(ErrorMessages.ERROR_MESSAGE_NUMBER_OF_RESULTS_VALUE_INVALID);
     }
 
     protected String getValidQuery(RequestInfo requestInfo) throws BadRequestException {
         return getQueryParam(requestInfo, QUERY)
             .filter(this::isValidQuery)
             .map(UriUtils::escapeWhiteSpace)
-            .orElseThrow(() -> new BadRequestException(ERROR_MESSAGE_QUERY_MISSING_OR_HAS_ILLEGAL_CHARACTERS));
+            .orElseThrow(() -> new BadRequestException(ErrorMessages.ERROR_MESSAGE_QUERY_MISSING_OR_HAS_ILLEGAL_CHARACTERS));
+    }
+
+    protected String getValidDepth(RequestInfo requestInfo) throws BadRequestException {
```

Comment:
Is this gonna be shared or is it specific for Organization query?

Reply:
Same as above